packages/interactive/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: ">=2.18.0 <3.0.0"
 
 dependencies:
-  analyzer: ^5.0.0
+  analyzer: 5.2.0
```

Comment:
what about sth like
  analyzer: >=5.0.0 <=5.2.0

Reply:
Whatever you think is best. I just hacked in the first thing that would get it to work again. :/